src/site/content/en/blog/better-youtube-web-part1/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Building a Better Web - Part 1: A faster YouTube on web'
+subhead: >
+  A case study of changes the YouTube Web team made to improve performance, increase their Core Web Vitals pass rates and lift key business metrics.
+description: >
+  This is Part 1 of our new series on Building a Better YouTube Web called "Building a faster YouTube on web"
+layout: post
+date: 2022-10-19
+authors:
+  - addyosmani
+  - sriramkrishnan
+hero: 'image/1L2RBhCLSnXjCnSlevaDjy3vba73/CPrVgVl7t1MuwWpXzvCE.jpg'
+alt: >
+  A hero image of the YouTube mobile web loading experience.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - case-study
+  - performance
+  - web-vitals
+---
+
+The Chrome team often talks about "building a *better* web", but what does that
+mean? Web experiences should be [fast](/why-speed-matters/),
+[accessible](/accessibility/), and leverage [device
+capabilities](/reliable/) in the moment when users need it most.
+Dogfooding is part of Google's culture, so the Chrome team has partnered with
+YouTube to share lessons learned along the way in a new series called, "Building
+a better web". The first part of the series will dive into how YouTube built a
+faster web experience.
+
+<figure>
+{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/hIcVf8Pob6DaLrmNjdNX.png", alt="PageSpeed Insights showing the Chrome UX Report data for YouTube Mobile Web passing the Core Web Vitals", width="800", height="450" %}
+  <figcaption>
+    The YouTube for mobile web Watch page passing the <a href="/vitals">Core Web Vitals</a> thresholds 
+  </figcaption>
+</figure>
+
+## Building a faster web
+
+Performance to YouTube is how fast videos and other content, such as
+recommendations and comments, load on web pages. It is also how quickly it
+responds to user interactions such as search, player control, likes, shares, and
+many others.
```

Comment:
```suggestion
At YouTube, _performance_ relates to how fast videos and other content—such as
recommendations and comments—load on web pages. Performance is also measured by how quickly YouTube
responds to user interactions such as search, player control, likes, and shares.
```

Reply:
This was a confusing paragraph with two "it" referring to different things in the second sentence. I think this reads better.